src/main/kotlin/net/irisshaders/lilybot/extensions/util/PublicUtilities.kt

```diff
@@ -33,16 +33,22 @@ class PublicUtilities : Extension() {
 	override val name = "public-utilities"
 
 	override suspend fun setup() {
-		val onlineLog = kord.getGuild(TEST_GUILD_ID)?.getChannel(ONLINE_STATUS_CHANNEL) as GuildMessageChannelBehavior
-
 		/**
-		 * Online notification
+		 * Online notification.
 		 * @author IMS212
 		 */
+		val onlineLog = kord.getGuild(TEST_GUILD_ID)?.getChannel(ONLINE_STATUS_CHANNEL) as GuildMessageChannelBehavior
 		responseEmbedInChannel(
 			onlineLog, "Lily is now online!", null, DISCORD_GREEN, null
 		)
 
+		/**
+		 * Thread data cleanup call.
```

Comment:
This is just the function title, could maybe add some information on why you're calling it here. Also, you're fully aware this will be run at startup only, right?

Reply:
We don't restart as often as we used to, so this may become an issue

Reply:
aight cool 👍🏼